content/docs/2.14/operate/events.md

```diff
@@ -88,6 +95,31 @@ Here is an overview of the supported destinations:
       uri: http://foo.bar  #An http endpoint that can receive cloudevent
 ```
 
+#### Azure Event Grid
+```yaml
+  destination:
+    azureEventGrid:
+      endpoint: Endpoint:foo.bar #endpoint from AzureEventGrid Topic
+```
+```TriggerAuthentication``` is needed to provide accesskey or identity for Azure Event Grid authentication. ```authenticationRef``` should be configured when sinking to Azure Event Grid
+
+**Connection String Authentication:**
+
+- `accessKey` - Access key string for the Azure Event Grid connection auth.
```

Comment:
To verify - `accessKey` is the expected key from TA, right?

Reply:
Yes. TA needs the `accessKey`. And KEDA will use it to auth with connection string.